Cargo.toml

```diff
@@ -34,3 +34,8 @@ web-sys = { version = "0.3" }
 chainhook-sdk = { git = "https://github.com/hirosystems/chainhook.git" }
 chainhook-types = { git = "https://github.com/hirosystems/chainhook.git" }
 stacks-codec = { path = "./components/stacks-codec" }
+
+# [patch.'https://github.com/stacks-network/stacks-core.git']
```

Comment:
We somestimes need to patch the stacks-core dependencies to work with local files instead of the git or crates.io versions).
Instead of re-writing this all the time, I think we can leave it in place but commented